content/en/blog/_posts/2017-03-00-K8Sport-Engaging-The-Kubernetes-Community.md

```diff
@@ -3,6 +3,8 @@ title: " The K8sPort: Engaging Kubernetes Community One Activity at a Time "
 date: 2017-03-24
 slug: k8sport-engaging-the-kubernetes-community
 url: /blog/2017/03/K8Sport-Engaging-The-Kubernetes-Community
+author: >
+   Ryan Quackenbush (Apprenda) 
 ---
 _Editor's note: Today’s post is by Ryan Quackenbush, Advocacy Programs Manager at Apprenda, showing a new community portal for Kubernetes advocates: the K8sPort._   
```

Comment:
I'd drop this note. We now just credit the person and optionally their organization.